app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    data = {}
+
+    def wrapper(*args) -> None:
```

Comment:
The return type of the `wrapper` function is specified as `None`, which is incorrect. The `wrapper` function should return the result of the `func` call, so the return type should match the return type of `func`. Consider changing the return type to match the expected return type of the decorated function.